src/neuvector/common/zarf.yaml

```
@@ -43,3 +43,7 @@ components:
                 name: neuvector
                 namespace: neuvector
                 condition: "'{.status.phase}'=Ready"
+          - description: Annotate Controller deployment for Neuvector SSO secret
+            cmd: |
+              SSO_CHECKSUM=$(./zarf tools kubectl get secret neuvector-secret -o json -n neuvector | sha256sum | cut -d' ' -f1)
+              ./zarf tools kubectl patch deployment neuvector-controller-pod -n neuvector -p "{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"neuvector-uds-secret\":\"$SSO_CHECKSUM\"}}}}}"
```

very nit but...
```suggestion
              ./zarf tools kubectl patch deployment neuvector-controller-pod -n neuvector -p "{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"checksum/uds-sso-secret\":\"$SSO_CHECKSUM\"}}}}}"
```
is a bit more in line with the upstream checksum annotation/standard ones I've seen.